pom.xml

```diff
@@ -739,6 +739,127 @@
             <scope>test</scope>
         </dependency>
 
+        <!-- Upper bound fix: list of transitive dependency to enforce highest version library -->
+        <dependency>
+            <groupId>com.jcraft</groupId>
+            <artifactId>jsch</artifactId>
```

Comment:
This is not a plugin so it should not be here. Whatever plugin added this dep improperly, either update to a version of that plugin which has itself been fixed to use 2.31+ POMs, or add an exclusion for this dep so we pick up the version from core.

Reply:
Ok, certain cases we can definitely do that, for others I will list them individually.